docs/idle-crafting.rst

```diff
@@ -45,9 +45,16 @@ needs to craft objects. Workshops that have a master assigned cannot be used in
 this way.
 
 When a workshop is designated for idle crafting, this tool will create crafting
-jobs and assign them to idle dwarves who have a need for crafting
-objects. Currently, bone carving and stonecrafting are supported, with
-stonecrafting being the default option. This script respects the setting for
-permitted general work orders from the "Workers" tab. Thus, to designate a
+jobs and assign them to idle dwarves who have a need for crafting objects. This
+script respects the setting for permitted general work orders from the "Workers"
```

Comment:
```suggestion
script respects the setting for permitted general work order labors from the "Workers"
```

Reply:
not to be confused with the "General work orders allowed" setting from the "Work orders" tab